syne_tune/optimizer/schedulers/neuralband/NeuralBand_UCB.py

```diff
@@ -0,0 +1,761 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Would it be possible to use lowercase for the name of the file (it is a python convention that we try to follow in all the code base): 
https://stackoverflow.com/questions/42127593/should-python-class-filenames-also-be-camelcased

PS: the same comment applies for other NeuralBand*.py files.

Reply:
sure, done.